ui/app/store/actions.js

```diff
@@ -47,41 +46,25 @@ const actions = {
   NETWORK_DROPDOWN_CLOSE: 'UI_NETWORK_DROPDOWN_CLOSE',
   showNetworkDropdown: showNetworkDropdown,
   hideNetworkDropdown: hideNetworkDropdown,
-  // menu state/
-  getNetworkStatus: 'getNetworkStatus',
   // transition state
   TRANSITION_FORWARD: 'TRANSITION_FORWARD',
-  TRANSITION_BACKWARD: 'TRANSITION_BACKWARD',
```

Comment:
Since this action has been deleted, the case handling it should be removed from the reducer as well. The same goes for the other actions.

Reply:
Strangely it seems that you've deleted the unit tests for the reducer, but left the relevant code there

Reply:
I've removed the reducer logic as well, I missed that